packages/@expo/cli/src/start/server/metro/createServerRouteMiddleware.ts

```diff
@@ -110,9 +110,7 @@ export function createRouteHandlerMiddleware(
         }
       },
       async handleRouteError(error) {
-        const { ExpoError } =
-          // TODO: Import from `@expo/server` after side effects are removed.
-          require('@expo/server/build/error') as typeof import('@expo/server/build/error');
+        const { ExpoError } = require('@expo/server') as typeof import('@expo/server');
```

Comment:
We can't ship this changed import until this is removed unfortunately: https://github.com/expo/expo/blob/main/packages/%40expo/server/src/install.ts

Reply:
Yes, the previous PR in the chain https://github.com/expo/expo/pull/38531 removes the side effects.

- https://github.com/expo/expo/pull/38531/files#diff-bcccdeb4f814d18796a06f42d7baea768f90a7a1647547e85c7634ff280300d8